.github/scripts/install_oneapi_bundle.bat

```diff
@@ -0,0 +1,56 @@
+REM Description: Install Intel Support Packages on Windows
+REM BKM reference: https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-5.html
+REM To-do: Add driver installation in this file.
+
+set ONEAPI_PARENT_DIR=C:\Program Files (x86)\Intel
+set INTEL_ONEAPI_PYTORCH_BUNDLE_URL=https://registrationcenter-download.intel.com/akdlm/IRC_NAS/5ca2021d-dd1a-4ab1-bd52-758fe63cf827/w_intel-for-pytorch-gpu-dev_p_0.5.2.19_offline.exe

```

Comment:
Let's update the bundle link to 0.5.3 and install pti also

Reply:
Updated the bundle link to 0.5.3. Then, I will try to add pti installation and sync with US val team about it.

Reply:
I guess we don't need sync, just follow the latest BKC. Anyway, you can submit another PR for it.

Reply:
Its just for effort saving. We do not need to do it for both side. I will also try to do it later with another PR for it.